lib/utils/nd_list.dart

```diff
@@ -427,7 +427,7 @@ class NDList<X> {
     final resolvedIndices = axis0Slices.expand((e) => e.parentIndices);
 
     return priorResult.resolveStep(resolvedIndices.toList(),
-        [axis0Slices.length, ...axis0Slices.first.shape.sublist(1)]);
+        [axis0Slices.length, ...axis0Slices.first.shape.sublist(0)]);
```

Comment:
`.sublist(0)` is equivalent to the list itself.

Reply:
Huh... The problem was that `axis0Slices.first.shape` was just `[1]`